### PR TITLE
Add environment variable

### LINKED ISSUE #234: Setup environment variables
We currently do not have `ENVIRONMENT` variable in our configuration, and it should be added.

### PR DESCRIPTION
**Pull request by anniegiang**
Closes #234

### CODE REVIEW

File: backend/config.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const configData = envSchema({
     .prop("AUTH_DOMAIN", S.string().required())
     .prop("AUTH_SECRET_KEY", S.string().required())
     .prop("AUTH_STATE", S.string().required())
+    .prop("ENVIRONMENT", S.string().default("dev").required())
```
Comment (mushketyk) [inline backend/config.js:14]: I would avoid setting a default value for an environment. If it is not defined, that would seem like an error that we should know about and fix it.

**Comment by mannykary** [comment]
Does this also need to be set in the deployment to the `review` and `staging` environments?

**Comment by mannykary** [comment]
Also, another thought, how is this different from `NODE_ENV`?

**Comment by anniegiang** [comment]
I believe `NODE_ENV` and `ENVIRONMENT` are the same (at least in meaning). Did we have a variable for `NODE_ENV` already? To be consistent should we stick with `NODE_ENV ` ?

**Comment by mannykary** [comment]
I think `NODE_ENV` is more standard. I do see it being used in our repo in a few places.

**Pull request merged**